Add Windows Development Guide

charlieJ107 opened the pull request:
Migrate Windows Developer Guide from orca-lite.

yangw1234 commented on python/nano/Windows_Development_Guide.md:
```diff
@@ -0,0 +1,44 @@
+# Windows development guide
```
Let's rename it to "Windows User Guide (WSL2)"?

yangw1234 commented on python/nano/Windows_Development_Guide.md:
```diff
@@ -0,0 +1,44 @@
+
+## Step 4: Install BigDL Nano from Pypi
+
+You can install BigDL nano from Pypi with `pip`.
```
Let's use different commands for pytorch and tensorflow

yangw1234 commented on python/nano/Windows_Development_Guide.md:
```diff
@@ -0,0 +1,56 @@
+# Windows User Guide (WSL2)
```
Also file name

yangw1234 commented on python/nano/Windows_Development_Guide.md:
```diff
@@ -0,0 +1,56 @@
+
+## Step 1: Install WSL2
+
+BigDL-Nano supports windows systems through *Windows Subsystem for Linux 2* (WSL2). 
```
For Windows, we recommend using *Windows Subsystem for Linux 2* (WSL2) to run BigDL-Nano.

yangw1234 commented on python/nano/Windows_Development_Guide.md:
```diff
@@ -0,0 +1,56 @@
+
+BigDL-Nano supports windows systems through *Windows Subsystem for Linux 2* (WSL2). 
+
+To install WSL2, you can now install everything you need by entering this command in an administrator PowerShell or Windows Command Prompt and then restarting your machine.
```
Specify windows version requirements.

yangw1234 commented on python/nano/Windows_Development_Guide.md:
````diff
@@ -0,0 +1,56 @@
+You can install BigDL nano from Pypi with `pip`.
+
+```
+pip install bigdl-nano
````
let's remove this.

charlieJ107 commented:
Updated in https://github.com/intel-analytics/BigDL/pull/3842/commits/9b88df79f6e03781df9ffaf5c8fb9322e15f05e6

* Removed the `pip install bigdl-nano` command
* Specificed the Windows version requirements
* Rename the file

yangw1234 commented:
LGTM
